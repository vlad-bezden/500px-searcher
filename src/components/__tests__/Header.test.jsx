import React from 'react'
import { shallow } from 'enzyme'

import Header from '../Header'

test('Header renders correctly', () => {
  const component = shallow(
    <Header src='http://test.com/test.jpg' />
  )
  const tree = component.getNode()

  expect(tree).toMatchSnapshot()
})