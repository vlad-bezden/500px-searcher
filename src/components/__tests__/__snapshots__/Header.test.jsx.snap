// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders correctly 1`] = `
<nav
  className="navbar navbar-default"
>
  <div
    className="navbar-header"
  >
    <a
      className="navbar-brand"
      href="#"
    >
      500px Searcher
    </a>
    <ul
      className="nav navbar-nav"
    >
      <li
        className="active"
      >
        <a
          href="#"
        >
          Search
        </a>
      </li>
    </ul>
  </div>
</nav>
`;
